### DISCUSSION TITLE
[Suggestion] Addition to docs.md - Installation macOS

### DISCUSSION THREAD

**Discussion by DrPython3** [vlang/v#12140] [General]
Hello folks!

I just stumbled about the [installation info in the docs.md](https://github.com/vlang/v/blob/master/doc/docs.md#linux-macos-freebsd-etc) for macOS users while installing **vlang** on Big Sur. The hints for installing from source are pretty ok. But, I'd like to suggest a tiny addition:

Most macOS users, regardless whether they are developers or just interested in software development, will have **brew** ([homebrew](https://brew.sh)) installed on their machines. And **vlang** is available as **homebrew formulae** ([see here](https://formulae.brew.sh/formula/vlang)).

Using brew for the installation is pretty easy and fast. Therefore it might be useful to add that installation method as a hint to the docs.md.

Thank you in advance for considering this and keep up your great work!

Best regards!

**Comment by spytheman** [comment]
Please do NOT install V from homebrew.

V is under development, and it is much easier (for both the users and for V's maintainers) and more supported, if you install it from source, and then update it with `v up`, instead of relying on a formula that may or not be updated, and that will leave you with an old V executable, that may have problems that are already solved in the newer versions.

That will change after V reaches version 1.0, but until then, all package managers do is create confusion for users and more work for V's developers.
